src/components/Icons.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const CartIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    className="h-6 w-6 "
+    fill="none"
+    viewBox="0 0 24 24"
+    stroke="currentColor"
+    strokeWidth={2}
+  >
+    <path
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      d="M16 11V7a4 4 0 00-8 0v4M5 9h14l1 12H4L5 9z"
+    />
+  </svg>
+);
+
+export const HumanLoggedIn = () => (
```

Comment:
Well, you've successfully incorporated an avatar icon, which is a positive step! However, there are a few adjustments needed to align it more closely with the design:

- The border surrounding the avatar icon doesn't quite match the design's specifications. Please consider updating the border to align with the intended design style such as border, width, height, and padding.

- Additionally, both the color of the icon itself and the color of the border need some adjustment to better match the design. Striving to match these colors as closely as possible will help achieve a more consistent and visually appealing result.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/1b3e4908-9140-4a2c-a22a-39bf5bcbe202) | ![image](https://github.com/AliRezaBmeDu/bookstore/assets/55840999/c55c440c-d0a3-41a2-8589-36cbd12be2cd) |

Your efforts to fine-tune these aspects of the avatar icon will contribute to a more polished and cohesive design. Keep up the good work! 🍔🎨